app/src/components/Editor.tsx

```diff
@@ -150,49 +177,123 @@ function Document() {
   );
 }
 
-const ParagraphContainer = styled.div``;
+const DocumentTitle = styled.h1`
+  text-align: left;
+  font-weight: normal;
+  font-size: 20px;
+  grid-column-start: 2;
+`;
+function FileNameDisplay({ path }: { path: string }) {
+  const extension = extname(path);
+  const base = basename(path, extension);
+
+  return (
+    <DocumentTitle>
+      {base}
+      <span style={{ fontWeight: 'lighter' }}>.{extension}</span>
```

Comment:
`extname` produces an extension which includes the `.` already
```suggestion
      <span style={{ fontWeight: 'lighter' }}>{extension}</span>
```